docs/api/extras/core/Path.html

```diff
@@ -1,123 +1,125 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
I have no idea... I changed only a few lines.
Would you like me to redo this change?